chain/polkadot/polkadot_chain.go

```diff
@@ -700,7 +703,19 @@ func (c *PolkadotChain) BuildRelayerWallet(ctx context.Context, keyName string)
 // SendFunds sends funds to a wallet from a user account.
 // Implements Chain interface.
 func (c *PolkadotChain) SendFunds(ctx context.Context, keyName string, amount ibc.WalletAmount) error {
-	panic("[SendFunds] not implemented yet")
+	// If denom == polkadot denom, it is a relay chain tx, else parachain tx
+	if(amount.Denom == c.cfg.Denom) {
```

Comment:
nit,style: This line (and some other if statements) are not `go fmt`ed. The parens are also unnecessary. 

The following is idiomatic. 

```go
if amount.Denom == c.cfg.Demon {
```

I am disappointed CI is not catching this `go fmt` error. 

I am curious what running `go fmt ./...` does?

Reply:
Ran go `fmt ./...` and included the proper formatting in the last commit